CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 - Images, videos, and other non-text content is now wider in View Mode
 - Notes may now be deleted directly from the history page
 - CodiMD instances can now be branded either with a '@ <custom string>' or '@ <custom logo>' after the CodiMD logo and text
+- The CodiMD frontend supports now optional image proxies defined in the backend.
```

Comment:
```suggestion
- The CodiMD frontend now supports optional image proxies defined in the backend.
```

Reply:
I would remove the word "now" here. Keep in mind, this changelog will be around in the 10th future release. Relative time units like now/soon/recently are not overly future proof.

Reply:
Changed that in 4fa8e770. Does it sound better with the change?

Reply:
I would write something like "Images will be loaded via proxy if an image proxy is configured in the backend".
Because you don't have to mention "The CodiMD frontend". We're already in the frontend :yum:

Reply:
Should be done now